tests/integrations/stdlib/test_httplib.py

```diff
@@ -4,13 +4,17 @@
 import pytest
 
 try:
+    # py3
```

Comment:
I've seem people argue for `if PY3K:` kind of variations for these to make this clearer.  Anyway, this is fine by me - i don't know if we have internal coding standards which have other opinions.

Reply:
Yeah, that might be better, but for now my goal was only to mark which was which.